main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
-cd $HOME
-agoric init --dapp-template dapp-card-store demo # use `agoric init --dapp-template dapp-card-store $DIRNAME` with any name you like
-cd demo
-agoric install community-dev # will take a minute to install all dependencies
-```
+Next, run the command:
+`source ~/.bashrc`
 
-Learn more about the [available Dapp templates](../dapps/dapp-templates.md).
+![009](./assets/009.png)
 
-## Start Agoric Solo Client and Simulated Blockchain
+Finally, install NVM with the command:
+`nvm install v18.16.0`
 
-```sh
-# Terminal 1
-agoric start --verbose --reset # `agoric start --reset` to start over
-```
+![010](./assets/010.png)
+
+
+# Installing Yarn
+Run the `corepack enable` command.
+`corepack enable`
 
-Leave this process and its logs running in its own terminal window.
+![023](./assets/023.png)
 
-## Open the Agoric Wallet and REPL
+Now run the `yarn –version` command
+`yarn –version`
 
-```sh secondary style2
-# Terminal 2
-cd demo
-agoric open --repl
+![049](./assets/049.png)
+
+
+# Installing Docker
+Now you'll install Docker using the two commands below. This first command will add the Docker GPG keys to your system, then add the repository to Apt for installation.
+```
+# Install Docker
+# Add Docker's official GPG key:
+sudo apt-get update
+sudo apt-get install ca-certificates curl gnupg
+sudo install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+sudo chmod a+r /etc/apt/keyrings/docker.gpg
+
+# Add the repository to Apt sources:
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
 ```
+![027](./assets/027.png)
 
-This should automatically open [http://127.0.0.1:8000](http://127.0.0.1:8000) in a new browser window or tab.
+Now you can install Docker!
+`sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin`
 
-To begin using the wallet, click the "Connect Solo Wallet" button.
+![028](./assets/028.png)
 
-![agoric wallet solo connection](./assets/agoric-open-repl-1.png)
+Now that Docker has been installed you'll need to add your user account to the Docker group.
+`sudo usermod -aG docker $USER`
 
-After your solo wallet is connected, then you're ready to deploy the contract and API. 
+![029](./assets/029.png)
 
-## Deploy the Contract and API
+Now go ahead and reboot your image.
 
-In our second terminal, deploy the contract to the simulated blockchain
-and the API to the solo client.
+![030](./assets/030.png)
 
-```sh secondary style2
-# Terminal 2
-cd demo # if not already there
-agoric deploy ./contract/deploy.js 
-agoric deploy ./api/deploy.js
-```
+Once your image has rebooted and you've logged back on, test that Docker works by running the `hello-world` sample.
```

Comment:
Confirming that docker is working is worthwhile. But let's not mix rebooting in there.

```suggestion
Confirm that Docker works by running the `hello-world` sample.
```

Reply:
Adding the user to the docker group in the last step sudo usermod -aG docker $USER requires a reboot, otherwise the new group membership isn't evaluated and the docker run hello-world (and all subsequent docker commands) will fail. Open to any better approaches though...